package.json

```diff
@@ -40,6 +40,6 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=6.0.0"
```

Comment:
@bcoe this doesn't look right, do we need to update the synthtool templates?

Reply:
I *think* we should ignore `package.json` when generating templates; since we customize it a bunch and the auto-generated one is stale, see:

https://github.com/googleapis/nodejs-web-risk/blob/master/synth.py#L30